src/applications/static-pages/cta-widget/components/messages/SignIn.jsx

```diff
@@ -14,9 +18,8 @@ const SignIn = ({
     headerLevel,
     alertText: (
       <p>
-        Try signing in with your <strong>DS Logon</strong>,
-        <strong>My HealtheVet</strong>, or <strong>ID.me</strong> account. If
-        you don’t have any of those accounts, you can create one now.
+        Sign in with your existing <ServiceProvidersText isBold /> account.{' '}
```

Comment:
We may want to run this past content? Sorry for the pings if it already has been 🙂 CC @RLHecht @DanielleThierryUSDSVA

Reply:
![image](https://user-images.githubusercontent.com/12773166/144648773-8c72bb1d-612f-4f88-a227-620ec5e5049a.png)

This is what will be in place of the `<ServiceProvidersList>` fysa CC @RLHecht @DanielleThierryUSDSVA

Reply:
Thank you @kelsonic 👍 .  There is no rush on this I think the Login.gov launch is being pushed back a bit.

Reply:
Related slack thread: https://dsva.slack.com/archives/C52CL1PKQ/p1638553654346700